src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -1,39 +1,29 @@
 @echo off
 setlocal
 
-set PROJECT_DIR=%~1
-set DOTNET_CMD=%~2
-set TOOLRUNTIME_DIR=%~3
-IF [%BUILDTOOLS_TARGET_RUNTIME%]==[] set BUILDTOOLS_TARGET_RUNTIME=win7-x64
+set DOTNET_CMD=%~1
+set TOOLRUNTIME_DIR=%~2
 set BUILDTOOLS_PACKAGE_DIR=%~dp0
-set PORTABLETARGETS_VERSION=0.1.1-dev
 set ROSLYNCOMPILERS_VERSION=1.2.0-beta1-20160202-02
```

Comment:
Unfortunately I don't have any good ideas here which is why I would really like to eliminate this entire custom publishing step. For now I guess we will just have to live with it.